src/pybind/mgr/dashboard_v2/controllers/perf_counters.py

```diff
@@ -21,6 +24,12 @@ def _get_latest(self, daemon_type, daemon_name, stat):
             return data[-1][1]
         return 0
 
+    def _unit_to_str(self, unit):
```

Comment:
We don't need that. as soon as this is rebased, we have this function in `src/pybind/mgr/mgr_module.py